examples/04.MQTT/mqtt.cc

```diff
@@ -114,6 +112,9 @@ void __cheri_compartment("mqtt_example") example()
 	auto heapAtStart =
 	  heap_quota_remaining(STATIC_SEALED_VALUE(mqttTestMalloc));
 
+	Debug::log("Generating client ID...");
+	mqtt_generate_client_id(clientID, clientIDLength);
```

Comment:
It would be nicer here to have a prefix and a random suffix. The same API can be used, just pass the tail of the string rather than the whole thing.